app/workers/asm_signup_promotion_worker.rb

```diff
@@ -0,0 +1,22 @@
+class AsmSignupPromotionWorker
+  include Sidekiq::Worker
+
+  def perform(account_id)
+    return if ENV['ASM_SIGNUP_BOUNTY'].nil?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.